### ISSUE TITLE
Snap deamon is polluting syslog

### ISSUE THREAD

**Issue by Kwaadpepper** [AdnanHodzic/auto-cpufreq#53]
![Capture d’écran de 2020-04-13 13-06-25](https://user-images.githubusercontent.com/6019313/79116022-a75a6e80-7d87-11ea-9c65-6a8bc9d2ec4f.png)

Is there anyway to silent the deamon ?

**Comment by AdnanHodzic** [comment]
Many new changes have been implemented as part of 1.2 release and I believe your problem is solved. 

Install latest version using snap package:

```
sudo snap install auto-cpufreq
```

or pull the latest changes and run:

```
sudo ./auto-cpufreq-installer
```

In case this doesn't solve your problem, please feel free to re-open this issue.

**Comment by Kwaadpepper** [comment]
Sorry to say that this is not solved.
#81 
I have posted my syslog

**Comment by AdnanHodzic** [comment]
Seems like this issue is identical what was reported in #82

**Comment by Red-Eyed** [comment]
Any news about that?
I had to disable auto-cpufreq as it prints too much to my syslog
I thin that we should increase the priority of this task

**Comment by AdnanHodzic** [comment]
I'm swamped IRL, but I'll do my best to take a look at this tonight.

Seems like this issue is only present on snap environemnts, as I couldn't replicate the problem when `auto-cpufreq` was installed using `auto-cpufreq-install` installer.

**Comment by AdnanHodzic** [comment]
Released a fix as part of 1.2.3 version which is available in beta channel, install it by running:

```
sudo snap install auto-cpufreq --beta
```

After you confirm it's working as expected for you I'll promote it to stable channel, merge changes with master and close this issue.

**Comment by Kwaadpepper** [comment]
Thanks I'll check this ASAP.

**Comment by AdnanHodzic** [comment]
Fix has already been confirmed in #82. Please feel to re-open if the problem persists. Changes have also been promoted to stable channel, so in case you switched to beta, you can switch back to stable by running:

```
sudo snap switch --channel=stable auto-cpufreq
```
